bundler/lib/dependabot/bundler/file_fetcher.rb

```diff
@@ -26,7 +26,9 @@ def self.required_files_message
       def ecosystem_versions
         {
           package_managers: {
-            "bundler" => Helpers.detected_bundler_version(lockfile)
+            bundler: {
+              "max" => Helpers.detected_bundler_version(lockfile)
```

Comment:
Today this only returns the major version, but long term this should be coerced as well to major.minor... given that long term once we're only running Bundler 2 we'll be returning the full `bundler` version extracted from the `BUNDLED_WITH` statement.